packages/mocks/http.js

```diff
@@ -6,22 +6,68 @@ const VERSION = Symbol.for('msw-integration-layer::version');
  * and it's import should be pure and non-sideeffectful.
  */
 export const http = {
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   get: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'get', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   post: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'post', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   put: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'put', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   patch: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'patch', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   delete: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'delete', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   options: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'options', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
+  all: (endpoint, handler) => {
```

Comment:
It was a conscious decision to _not_ add `all`, because its magic-y. Our current mocking solution has tons of magic, and we're trying to get rid of magic and have people be explicit in what they're trying to mock; so also their http methods.

Reply:
makes sense 👍

Reply:
removed